### ISSUE TITLE
Inconsistent / Incorrect TypeScript Build Error reporting for very large file

### ISSUE THREAD

**Issue by foyzulkarim** [Microsoft/TypeScript#16606]
**TypeScript Version:**  2.3.4 

**Code**
Gist can be found here
https://gist.github.com/foyzulkarim/b23b84810d3277345ee6e752dfdc7fe8

As you can see, the above file is 435 lines long. When I Build the solution (in Visual Studio 2017), it tells me that 
**"tsc.exe" exited with code 1**
But if I remove the a big chunk, approx. more than 100 lines, from the file, Visual Studio builds the project smoothly, without any problem.  

```ts
Severity	Code	Description	Project	File	Line	Suppression State
Error		"tsc.exe" exited with code 1.	WebClient			
```

**Expected behavior:**
Error should complain about the file length, or inform us to split the configuration file in multiple smaller files. 

**Actual behavior:**
Error message tells 	"tsc.exe" exited with code 1.	which is not correct. 
![image](https://user-images.githubusercontent.com/497812/27262403-f81458d6-5477-11e7-80b5-b8f09a4d72a0.png)

**Comment by rbuckton** [comment]
This sounds like it might be a compiler crash. Is there any additional information in the build output in Visual Studio associated with this?

**Comment by DanielRosenwasser** [comment]
This also doesn't appear to be self-contained. If you have a project that reproduces the error that would help us diagnose the problem a little more easily.

**Comment by foyzulkarim** [comment]
Thanks. I will make  a project in GitHub soon and share it with you here. Just to let you know that, I was discussing with one of my colleague about this, and he confirmed he faced this problem too. then he split up the config file into multiple files, and the project compiles well. So, I am confident about this issue. 
Thanks for your response.

**Comment by mhegazy** [comment]
Automatically closing this issue for housekeeping purposes. The issue labels indicate that it is unactionable at the moment or has already been addressed.
